src/amd_layout.js

```diff
@@ -1,9 +1,16 @@
 (function(global, factory) {
-  if (typeof define === 'function' && define.amd)
+  if (typeof module === "object" && typeof module.exports === "object" )
+    module.exports = global.document
+      ? factory(global, true)
+      : function(w) {
```

Comment:
This is the part that I don't understand. If `global.document` is present, exports will basically evaluate to `module.exports = Zepto`. Nothing wrong with that. But the other code path, this one, exports a function that accepts what I presume is a window object, and *returns* the result of `factory(w)`, which would be Zepto. These are entirely different exports, however. How do you justify this?